packages/automerge-repo-react-hooks/src/useBootstrap.ts

```diff
@@ -68,16 +69,22 @@ export const useBootstrap = ({
       setHash(`${urlHashKey}=${documentId}`);
     if (localStorageKey) localStorage.setItem(localStorageKey, documentId);
   },
+  onInvalidDocumentId = (e) => createDocument,
 } = {}) => {
   const repo = useRepo();
   const hash = useHash();
 
   // Try to get existing document; else create a new one
   const handle = useMemo(() => {
     const existingDocumentId = getDocumentId(hash);
-    return existingDocumentId
-      ? repo.find(existingDocumentId)
-      : createDocument(repo, onCreate);
+    try {
+      return existingDocumentId
+        ? repo.find(existingDocumentId)
+        : createDocument(repo, onCreate);
+    } catch (e) {

```

Comment:
I'm assuming `repo.find` will throw on invalid document ID

Reply:
it will ... soon!